changelog/@unreleased/pr-534.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: fix
+fix:
+  description: '[Gradle JDK Enablement] Mingw is not supported'
```

Comment:
Perhaps say something like 'Fallback to normal when using `mingw`' or replace `normal` with whatever it needs to be, idk